src/services/expenses.service.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable function-paren-newline */
+const { Expense } = require('../models/Expense.model');
+const Match = require('../filterExp');
+
+const allExpenses = async (id, categories, from, to) => {
+  const fromDate = from
+    ? new Date(from).toISOString()
+    : new Date(0).toISOString();
+
+  const toDate = to ? new Date(to).toISOString() : new Date().toISOString();
+  const result = await Expense.findAll();
+
+  return result.filter((expense) =>
+    Match.isExpenseMatch(expense, id, categories, fromDate, toDate),
+  );
+};
+
+const expenseById = async (id) => {
+  return Expense.findByPk(id);
+};
+
+const createExpenses = async ({
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note,
+}) => {
+  const id = Math.floor(Math.random() * 10000);
```

Comment:
- You can set autoincrement parameter, to your id property so you don't need to set id manually.
- And this way to generate id is bad, as they can be duplicated

Reply:
> * You can set autoincrement parameter, to your id property so you don't need to set id manually.
> * And this way to generate id is bad, as they can be duplicated

Not fixed